keycloak/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM jboss/keycloak:16.0.0
+
+COPY ./micro2do-realm.json /tmp/realm-export.json
```

Comment:
it better to use docker compose and mound these files and don't spend time building a docker image

Reply:
Moved keycloack configuration to docker-compose